l10n/configs/vendor.toml

```diff
@@ -0,0 +1,47 @@
+# All content handled by the vendor or Mozilla staff should
+# be covered by the [[paths]] in this file, with the relevant locales.
+
+basepath = ".."
+
+locales = [
+    "cs",
+    "de",
+    "en-CA",
+    "en-GB",
+    "es-AR",
+    "es-CL",
```

Comment:
`es-CL` wasn't on the lists of Spanish, @gueroJeff, was that a mistake or intentional?